impl/src/test/java/org/ehcache/impl/internal/store/tiering/CacheStoreTest.java

```diff
@@ -63,28 +71,36 @@
  */
 public class CacheStoreTest {
 
+  @Mock
+  private CachingTier<Number, CharSequence> numberCachingTier;
+  @Mock
+  private AuthoritativeTier<Number, CharSequence> numberAuthoritativeTier;
+  @Mock
+  private CachingTier<String, String> stringCachingTier;
+  @Mock
+  private AuthoritativeTier<String, String> stringAuthoritativeTier;
```

Comment:
If this is just eliminating doing the mocking in the first two lines of each test, then consider my vote a -1 against this specific change... just a vote however.  Would like others to weigh in.